src/components/Label.tsx

```diff
@@ -1,19 +1,16 @@
-/* eslint-disable react-native/no-unused-styles */
 import React from 'react';
-import { View, Text, StyleSheet } from 'react-native';
+import { View, Text, StyleSheet, TextProps } from 'react-native';
 
-import { palette, typography } from 'app/styles';
-
-type LabelType = 'warning' | 'success' | 'error' | 'neutral';
+import { typography } from 'app/styles';
 
 interface Props {
-  type: LabelType;
-  children: React.ReactNode;
+  additionalLabelStyle: TextProps;
```

Comment:
maybe without `additional`? By convention it's always additional ;)

is `TextProps` even valid valid here?

Also, you can use `StyleProp<TextStyle>`